assets/app/scripts/controllers/builds.js

```diff
@@ -58,6 +58,72 @@ angular.module('openshiftConsole')
       }      
     };
 
+    // Function which will 'instantiate' new build from given buildConfigName
+    $scope.startBuild = function(buildConfigName) {
+      var req = {metadata:{name:buildConfigName}};
+      DataService.create("buildConfigs/instantiate", buildConfigName, req, $scope).then(
+        function(build) { //success
+            $scope.alerts = [
+            {
+              type: "success",
+              message: "Build " + build.metadata.name + " has started.",
+            }
+          ];
+        },
+        function(result) { //failure
+          $scope.alerts = [
+            {
+              type: "error",
+              message: "An error occurred while starting the build.",
+              details: "Status: " + result.status + ". " + result.data,
+            }
+          ];
+        }
+      );
+    };
+
+    // Function which will 'clone' build from given buildName
+    $scope.cloneBuild = function(buildName) {
+      var req = {metadata:{name:buildName}};
+      DataService.create("builds/clone", buildName, req, $scope).then(
+        function(build) { //success
+            $scope.alerts = [
+            {
+              type: "success",
+              message: "Build " + build.metadata.name + " has been rerun from build " + buildName + ".",
```

Comment:
Maybe "Build newbuild has been rebuilt from oldbuild."  Since the action they selected was "Rebuild"

or possibly "Build oldbuild is being rebuilt as newbuild."  since that implies the action is still in progress, which it is.